ArduPlane/quadplane.cpp

```diff
@@ -3284,9 +3289,9 @@ void QuadPlane::update_throttle_mix(void)
         return;
     }
 
-    if (plane.control_mode == &plane.mode_qstabilize) {
+    if ((plane.control_mode == &plane.mode_qstabilize) || (plane.control_mode == &plane.mode_qhover) || (plane.control_mode == &plane.mode_qacro)) {
```

Comment:
This change doesn't seem related to the PR title and would appear to be a significant behavioural change?

Reply:
this is the QACRO bugfix; related because airmode still needs manual throttle mix when throttle is zero
I don't think this applies to qhover mode though; will remove it from the test.